### PR TITLE
Solutions to statistics tasks

### PR DESCRIPTION
**Pull request by pauleonix**

### CODE REVIEW

**Comment by pauleonix** [comment]
@XylotrupesGideon  Any thoughts on statistics task 5?

File: src/notebook_unit1.ipynb [hunk 1]
```diff
@@ -95,7 +95,7 @@
    "source": [
```
Comment (XylotrupesGideon) [inline src/notebook_unit1.ipynb:95]: <p>retruns</p><p><strong>No such file or directory: '../plots/expec_z.pdf</strong>'</p><p>i think it needs to create a plot folder first</p><p> </p><p>maybe at the start of the script already</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pauleonix/ssd_course/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (XylotrupesGideon) [inline src/notebook_unit1.ipynb:95]: <p>see above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pauleonix/ssd_course/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (XylotrupesGideon) [inline src/notebook_unit1.ipynb:95]: <p>Needs to add .csv to .gitignore</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pauleonix/ssd_course/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (XylotrupesGideon) [inline src/notebook_unit1.ipynb:95]: <pre class="ql-syntax" spellcheck="false">Line #2.    d = pd.DataFrame(
</pre><p>"d" is not a very explicit variable name I would prefer something like "euclid_dist" for readability</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pauleonix/ssd_course/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

**Comment by pauleonix** [comment]
@XylotrupesGideon any feedback to the `mkdir` part to create the `plots` directory? It's located in the cell which import all the used modules.

**Pull request merged**